lpr.WebAPI/Controllers/OrganisationController.cs

```diff
@@ -37,6 +37,21 @@ public class OrganisationController : ControllerBase {
       return StatusCode(200, org);
     }
 
+    [HttpGet("GetPackagesPaginated/{page}/{amount}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<IActionResult> GetOrganisationsPaginated(int page,
+                                                               int amount) {
+      try {
+        List<Organisation> output =
+            await _organisationService.GetOrganisationsPaginatedAsync(page,
+                                                                      amount);
+        return StatusCode(200, output);
```

Comment:
Can you add a parameter with the total amount of pages or the total amount projects? So we can make pagination in the fronted.

Reply:
This is yet unchanged due to the change of the way pagination works.